controllers/api/v1/search_test.go

```diff
@@ -0,0 +1,318 @@
+package apiv1
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/google/uuid"
+
+	_ "google-scraper/initializers"
+	. "google-scraper/tests"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("SearchController", func() {
+	Describe("POST /api/v1/search", func() {
+		Context("Given valid credentials", func() {
+			Context("given the params are valid", func() {
+				It("returns 201 status code", func() {
+					user := FabricateUser("John", "john@example.com", "secret")
+					client := FabricateOAuthClient(uuid.New().String(), uuid.New().String())
+					oauthToken := FabricateOAuthToken(client, user.Id)
+					filePath := AppRootDir() + "/tests/fixtures/shared/valid_keywords.csv"
+					_, body := CreateMultipartFormData(filePath)
+					mockResponseFilePath := AppRootDir() + "/tests/fixtures/services/crawler/valid_get_response.html"
+					MockCrawling(mockResponseFilePath)
+
+					header := http.Header{
+						"Authorization": {fmt.Sprintf("Bearer %v", oauthToken.GetAccess())},
+						"Content-Type":  {"multipart/form-data; boundary=multipart-boundary"}}
+
+					response := MakeAuthenticatedRequest("POST", "/api/v1/search", header, body, nil)
+
+					Expect(response.StatusCode).To(Equal(http.StatusCreated))
+				})
+
+				It("returns empty response", func() {
```

Comment:
As you want to return an empty response from upload csv request. Please consider using 204 instead of 201 (https://jsonapi.org/format/#crud-creating-responses-204)